src/message.ts

```diff
@@ -346,11 +407,28 @@ export class Message implements Sayable {
   // public to(room: Room): void
   // public to(): Contact|Room
   // public to(contact?: Contact|Room|string): Contact|Room|void {
+
   /**
-   * @todo document me
+   * Set the destination as Contact for the message
+   *
+   * @param {Contact} contact
+   * @returns {void}
    */
   public to(contact: Contact): void
```

Comment:
This method should be marked as `@private` because normally use should not use Wechaty API to set the destination of the message by call `to()`. 

We use `Sayable`s to send message to contact, like `contact.say()` and `room.say()`